### PR TITLE
Ana/3855 fix validators routing

### LINKED ISSUE #3855: Object.a
Sentry Issue: [FRONTEND-8T](https://monitoring.lunie.io:9000/lunie/frontend/issues/373/?referrer=github_integration)

```
Error: Non-Error exception captured with keys: extensions, locations, message, path
  at Object.a (/bundles/c31fcd422fdbfb1f8688748e08295f8af352cf03.js:45:128486)
  at None (/bundles/c31fcd422fdbfb1f8688748e08295f8af352cf03.js:74:51446)
  at Array.map (<anonymous>)
  at Object.next (/bundles/c31fcd422fdbfb1f8688748e08295f8af352cf03.js:74:51393)
  at w (/bundles/c31fcd422fdbfb1f8688748e08295f8af352cf03.js:65:10227)
...
(17 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3855 (this time for real)

**Description:**

There were a couple of broken routes lying around and also tried to fix that mysterious `uncaught Promise` popping up in sign ins and here to (from the SignIn/ Signup link to portfolio in ActionModal when you are not signed in.

Sadly this I wasn't able yet

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -649,7 +649,7 @@ export default {
       this.close()
 
       this.$store.dispatch(`signOut`, this.network)
-      if (this.$route.name !== `portfolio`) this.$router.push(`portfolio`)
+      if (this.$route.name !== `portfolio`) this.$router.push(`/`)
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue:652]:
```suggestion
      if (this.$route.name !== `portfolio`) this.$router.push({ name: 'portfolio' })
```

https://router.vuejs.org/guide/essentials/navigation.html

File: src/components/transactions/message-view/RestakeTxDetails.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
           transaction.details.from[0] | resolveValidatorName(validators)
         }} </router-link
       ><i class="material-icons notranslate arrow">arrow_right_alt</i>
-      <router-link :to="`staking/validators/${transaction.details.to[0]}`">
+      <router-link :to="`/staking/validators/${transaction.details.to[0]}`">
```
Comment (faboweb) [inline src/components/transactions/message-view/RestakeTxDetails.vue:18]:
```suggestion
      <router-link :to="{ name: 'validator', params: { validator: transaction.details.to[0] }}">
```

File: src/ActionModal/components/ActionModal.vue (hunk 1 above)
Comment (Bitcoinera) [reply src/ActionModal/components/ActionModal.vue:652]: Then we need to change a bunch of routes here

Comment (Bitcoinera) [reply src/ActionModal/components/ActionModal.vue]: OK, actually not 😅 There were two like this for extension. All good

**Comment by Bitcoinera** [comment]
Let's merge! E2E test failed because something weird

File: tests/unit/specs/components/transactions/message-view/__snapshots__/RestakeTxDetails.spec.js.snap [hunk 3]
```diff
@@ -33,7 +33,7 @@ exports[`RestakeTxDetails renders a restake transaction message 1`] = `
     </i>
      
     <router-link-stub
-      to="staking/validators/cosmosvaloper18ymm350peujvq2xy9ymyqj4v34ekvnk3tsekuz"
+      to="[object Object]"
```
Comment (jbibla) [inline tests/unit/specs/components/transactions/message-view/__snapshots__/RestakeTxDetails.spec.js.snap:36]: is this expected?

**Comment by jbibla** [comment]
how do you know the e2e is unrelated?

File: tests/unit/specs/components/transactions/message-view/__snapshots__/RestakeTxDetails.spec.js.snap (hunk 3 above)
Comment (Bitcoinera) [reply tests/unit/specs/components/transactions/message-view/__snapshots__/RestakeTxDetails.spec.js.snap:36]: seems so. Search for `to=[object Object]`. There are a bunch already like this in snapshots.

But thank you for pointing it out. It's interesting

**Comment by Bitcoinera** [comment]
> how do you know the e2e is unrelated?

Just checked by rerunning ✌️  But yeah, logs seemed completely unrelated too

**Pull request merged**